data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/JacksonSpan.java

```diff
@@ -50,10 +51,10 @@ public class JacksonSpan extends JacksonEvent implements Span {
     private static final String DROPPED_EVENTS_COUNT_KEY = "droppedEventsCount";
     private static final String LINKS_KEY = "links";
     private static final String DROPPED_LINKS_COUNT_KEY = "droppedLinksCount";
-    private static final String SERVICE_NAME_KEY = "serviceName";
-    private static final String TRACE_GROUP_KEY = "traceGroup";
+    public static final String SERVICE_NAME_KEY = "serviceName";
```

Comment:
If you are using these in the tests, make them package protected. Remove `public`.

Reply:
It needs to be public because it is used in data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoStandardCodec.java